lib/lambda/sinkMainProcessors.ts

```diff
@@ -421,7 +446,34 @@ export const insertNewSeatoolRecordsFromKafkaIntoMako = async (
 
       seatoolRecord.STATE_PLAN.SPW_STATUS_ID = await oneMacSeatoolStatusCheck(seatoolRecord);
 
-      const safeSeatoolRecord = opensearch.main.seatool.transform(id).safeParse(seatoolRecord);
+      let safeSeatoolRecord;
+      try {
```

Comment:
Wouldn't this have been caught my the parent `try/catch` on line 427? Should we handle this in the catch on line 505?